tests/expected_report.txt

```diff
@@ -33,7 +33,7 @@ tests/input/UselessConditions.php                     20      0
 ----------------------------------------------------------------------
 A TOTAL OF 238 ERRORS AND 0 WARNINGS WERE FOUND IN 27 FILES
 ----------------------------------------------------------------------
-PHPCBF CAN FIX 207 OF THESE SNIFF VIOLATIONS AUTOMATICALLY
+PHPCBF CAN FIX 202 OF THESE SNIFF VIOLATIONS AUTOMATICALLY
```

Comment:
Interesting, so `UselessIfConditionWithReturn` can't fix things anymore...

Reply:
See my comments below.

Reply:
It can still fix things where you use operators that produce boolean - comparisons etc. Otherwise it may have produced broken code.

Reply:
@Majkl578 it can still fix everything, there’s new option

Reply:
I know, but then it can produce broken code based on wrong assumptions. I think this is better way.

Reply:
```php
public function isEmpty() : bool
{
    return $this->count() ? true : false;
}
```
would become:
```php
public function isEmpty() : bool
{
    return $this->count(); // boom!
}
```